modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -90,6 +91,19 @@ export default function EditObjectFolder({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [objectFolderName]);
 
+	useEffect(() => {
+		if (showChangesSaved) {
+			setTimeout(() => {
+				dispatch({
+					payload: {updatedShowChangesSaved: false},
+					type: TYPES.SET_SHOW_CHANGES_SAVED,
+				});
+			}, 5000);
```

Comment:
hey @Artartur, I'm finding this disappearance time to be very long, have you received any instructions about this waiting time?

Reply:
Ok @Artartur! Thanks for this information!